server/public/model/system.go

```diff
@@ -119,6 +121,9 @@ type SupportPacket struct {
 	LicenseTo             string `yaml:"license_to"`
 	LicenseSupportedUsers int    `yaml:"license_supported_users,omitempty"`
 	LicenseIsTrial        string `yaml:"license_is_trial,omitempty"`
+	CustomerID            string `yaml:"customer_id"`
+	LicenseID             string `yaml:"license_id"`
+	ServerID              string `yaml:"server_id"`
```

Comment:
1/5 the name could be a bit more telling. Maybe `TelemetryID`

Reply:
Hey, the ask here is to standardize and it's called ServerID across telemetry.